examples/generative/text_generation_with_miniature_gpt.py

```diff
@@ -30,14 +30,30 @@
 """
 ## Setup
 """
-import tensorflow as tf
-from tensorflow import keras
-from tensorflow.keras import layers
-from tensorflow.keras.layers import TextVectorization
+# We set the backend to TensorFlow. The code works with
+# both `tensorflow` and `torch`. It does not work with JAX
+# due to the behavior of `jax.numpy.tile` in a jit scope
+# (used in `causal_attention_mask()`: `tile` in JAX does
+# not support a dynamic `reps` argument.
```

Comment:
I remember this issue. It's basically intractable apart from refactoring the example (I'm not sure the tile arg actually needs to be dynamic).